cxplat/src/tags.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+// Pool tags used by the cxplat library.
+#define CXPLAT_DEFAULT_TAG 'edxc'
```

Comment:
Thoughts on changing the naming pattern to be `CXPLAT_TAG_*`? It's easier to search for them that way too.

Reply:
Done